architecture/adr-006-skeleton-screens.md

```diff
@@ -0,0 +1,36 @@
+# Architecture Decision Record: Skeleton Screens
+
+## Context
+
+In the work of trying to improve the performance
```

Comment:
Totally nitpicking but I wonder why these lines of text are so short. Markdownlint enforces a maximum line length of 80 characters but many of these lines are <50 for no reason that I could tell :smile:.